Pod/Classes/CDTQQuerySqlTranslator.m

```diff
@@ -394,6 +413,18 @@ + (CDTQSqlParts *)wherePartsForAndClause:(NSArray *)clause usingIndex:(NSString
                           parameters:sqlParameters];
 }
 
++ (NSString *)placeholdersForInList:(NSArray *)values withParameters:(NSMutableArray *)sqlParameters
+{
+    NSMutableArray *inOperands = [NSMutableArray array];
+    for (NSObject *value in values) {
+        [inOperands addObject:@"?"];
+        [sqlParameters addObject:value];
+    }
+    
+    NSString *joined = [inOperands componentsJoinedByString:@", "];
+    return [NSString stringWithFormat:@"( %@ )", joined];
+}
```

Comment:
Updated to `updatingParameterValues` as the code is called in two different places.  Also changed `placeholdersForInList` to `placeholdersForList` based on later comments.  Done in b667c92d0cf9994ca7a8751af6e7efedcb6f3199.